GVFS/GVFS.Platform.Windows/WindowsRequiredCallbacks.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Windows.ProjFS;
+
+namespace GVFS.Platform.Windows
+{
+    public class WindowsRequiredCallbacks : IRequiredCallbacks

```

Comment:
This class seems like it is not needed.  Why not just have `WindowsFileSystemVirtualizer` implement `IRequiredCallbacks`?  This class just creates a bunch of stuff so that methods in `WindowsFileSystemVirtualizer` will get called anyways.  This class would completely go away and you would call
`this.virtualizationInstance.StartVirtualizing(this)` instead.

Reply:
That seems like a fair suggestion.  @wilbaker any objection with that approach?

Reply:
No objections from me, that approach sounds good.